content/guides/process.md

```diff
@@ -1,23 +1,80 @@
 ---
-title: Datadog Process Monitoring
+title: Datadog Process and Container Monitoring
 kind: guide
 listorder: 16
 beta: true
 ---
 
 ## Introduction
 
-The new Datadog Process Monitor allows for real-time process-level visibility.  After configuring the Agent, information from a host’s /proc file system will be streamed into Datadog and presented in a table at [https://app.datadoghq.com/process](https://app.datadoghq.com/process).
+Datadog Process and Container Monitoring allows for real-time visibility of the most granular elements in a deployment.  Taking inspiration from bedrock tools like `htop`, this centralized view, combined with existing tagging capabilities, allows you to understand what is going on at any level of your system and drill all the way down into the most fine details.
 
-The table can be searched and filtered, and processes can be aggregated on tags associated with the hosts that they are running on.
+{{< img src="process/live_process_preview.png" >}}
 
-Each row in the table can be expanded for more contextual information about an individual process.
+## Searching, Filtering, and Pivoting
 
-Checking “show summary graphs” at the top of the page will graph metrics for all processes that are shown in the table.
+**String Search**
 
-{{< img src="process/live_process_preview.png" >}}
+Processes and containers are by their nature extremely high cardinality objects.  Our fuzzy string search will match any facet of a process or container, giving you a view into exactly what you want.  Below is our Demo environment, filtered with the string `postgres /9.`.  Note that `/9.` has matched in the current working directory, and that `postgres` matches the command itself.
```

Comment:
> search will match any facet of a process or container

This is not actually true. For processes we only search on cmdline and user, for containers it's just name and image (and soon id). in your example both parts are in the full cmdline.